.github/pull_request_template.md

```diff
@@ -14,6 +14,19 @@ _Write out steps for how someone could test this PR against the acceptance crite
 
 _If relevant, e.g. for a front-end feature_
 
+## Definition of Done Checklist
+- [ ] UI works as designed
+- [ ] OESA: Code refactored for clarity
+- [ ] OESA: Dependency rules followed
+- [ ] Automated unit tests updated and passed
+- [ ] Automated integration tests updated and passed
+- [ ] Automated quality tests updated and passed
+- [ ] Automated load tests updated and passed
+- [ ] Automated a11y tests updated and passed
+- [ ] Automated security tests updated and passed
+- [ ] 90%+ Code coverage achieved
+
```

Comment:
wonder if we need a validation tasks?

Reply:
no worries @jonnalley, we discussed possibly adding the new Vest form validation as a DoD item in our ENG Hours meeting this week.